### PR TITLE
Packet drop release note

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

4.13+
Issue:

https://issues.redhat.com/browse/OSDOCS-6273 
Link to docs preview:

image
QE review:
- [X] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by Amoghrd** [comment]
LGTM

**Comment by Amoghrd** [comment]
Will wait for @msherif1234 to review too

File: networking/network_observability/network-observability-operator-release-notes.adoc [hunk 1]
```diff
@@ -13,6 +13,10 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../networking/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-packet-drop-1.4"]
+==== Packet drops
+In the 1.4 release of the Network Observability Operator, eBPF tracepoint hooks are used to enable packet drop tracking. You can now detect and analyze the cause for packet drops, and in turn, make decisions to optimize network performance. For more information, see xref:../../networking/network_observability/observing-network-traffic#network-observability-pktdrop-overview_nw-observe-network-traffic[Configuring packet drop tracking] and xref:../../networking/network_observability/observing-network-traffic#network-observability-packet-drops_nw-observe-network-traffic[Working with packet drops].
```
Comment (stevsmit) [inline networking/network_observability/network-observability-operator-release-notes.adoc:18]: Is in turn necessary here? 

```suggestion
In the 1.4 release of the Network Observability Operator, eBPF tracepoint hooks are used to enable packet drop tracking. You can now detect and analyze the cause for packet drops and make decisions to optimize network performance. For more information, see xref:../../networking/network_observability/observing-network-traffic#network-observability-pktdrop-overview_nw-observe-network-traffic[Configuring packet drop tracking] and xref:../../networking/network_observability/observing-network-traffic#network-observability-packet-drops_nw-observe-network-traffic[Working with packet drops].
```

Also check these xrefs; I don't see .adoc in them.

**Comment by stevsmit** [comment]
@skrthomas 1 comment, 2 nits

File: networking/network_observability/network-observability-operator-release-notes.adoc (hunk 1 above)
Comment (skrthomas) [reply networking/network_observability/network-observability-operator-release-notes.adoc:18]: @stevsmit ah, sorry, I noted this in my [main release notes](https://github.com/openshift/openshift-docs/pull/63934) but didn't note it here, but basically the asciidoc link will be broken until I merge the main feature PR, which I don't do until GA day. So my release notes are always broken </3 

Is in turn necessary - I guess it really isn't. Good pt.

**Pull request merged**